RFC/src/RFC-0300_DAN.md

```diff
@@ -1 +1,205 @@
+* The set of VNs that participate in managing state of a specific DA is called a [Committee]. A committee is selected during the asset
+issuance process and membership of the committee can be updated at [Checkpoint]s.
+* It is the VNs responsibility to ensure that every state change in a digital asset conforms to the contract's rules.
+* VNs accept digital asset [Instructions] from clients and peers. [Instructions] allow for creating, updating, and
+  expiring digital assets on the DAN.
+* VNs provide additional collateral when accepting an offer to manage an asset, which is stored in a multi-signature
+  contract on the base layer. This collateral can be taken from the VN if it is proven that the VN engaged in
+  malicious behaviour.
+* VNs participate in fraud proof validations in the event of consensus disputes (which could result in the malicious VN's
+  collateral being slashed).
+* Digital asset metadata (e.g. large images) are managed by VNs. The large data itself will not be stored on the VNs but an external location and a hash of the data can be stored. Whether the data is considered part of the state
+  (and thus checkpointed) or out of state depends on the type of digital asset contract employed.
+
+#### Fees
+Fees will be paid to VNs based on the amount of work they did during a checkpoint period. The fees will be paid from a fee pool which will be collected
+and reside in a UTXO that is accessible by the committee. The exact mechanism for the the payment of the fees by the committee and who pays the various
+types of fees is still under discussion.
+
+#### Checkpoints
+VNs periodically post checkpoint summaries to the [base layer] for each asset that they are managing. The checkpoints will form an immutable
+record of the [Digital Asset] state on the base-layer. There will be two types of checkpoints:
+* An Opening checkpoint (OC) will:
+  * Specify the members of the VN committee.
+  * Lock up the collateral for the committee members for this checkpoint period.
+  * Collect the fee pool for this checkpoint period from the Asset Issuer into a Multi-Sig UTXO under the control of the committee.
+  This can be a top-up of the fees or a whole new fee pool.
+
+* A Closing checkpoint (CC) will:
+  * Summarize the Digital Asset state on the base layer.
+  * Release the fee pay outs.
+  * Release the collateral for the committee members for this checkpoint period.
+  * Allow for committee members to resign from the committee
+
+After an DA is issued there will immediately be an Opening checkpoint. After a checkpoint period there will then be a Closing checkpoint followed
+immediately by an Opening checkpoint for the next period, we will call this set of checkpoints an Intermediate checkpoint. This will continue
+until the end of the asset's lifetime where there will be a final Closing checkpoint that will be followed by the retirement of the asset.
```

Comment:
I agree that this mechanism has the potential to create too much overhead. Reword to suggest both
O -- CO -- CO -- CO -- CO -- CO -- C
and 
O -- I -- I -- CO -- I -- I -- C
are possible candidates for checkpointing and that this is still under discussion.